.buildkite/commands/finalize-release.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -eu
+
+RELEASE_NUMBER=$1
+
+if [[ -z "${RELEASE_NUMBER}" ]]; then
+    echo "Usage $0 <release number>"
+    exit 1
+fi
+
+echo '--- :git: Configure Git for release management'
+.buildkite/commands/configure-git-for-release-management.sh
+
+echo '--- :git: Checkout release branch'
+# FIXME: Don't forget to revert to "release/" only
+.buildkite/commands/checkout-release-branch.sh "test-release/$RELEASE_NUMBER"
```

Comment:
Will need to rebase and amend this. Like it was in 9e4281dd4608c60412ce868b14b0ce2338b3920f